### DISCUSSION TITLE
How do I remotely open new tabs on librewolf?

### DISCUSSION THREAD

**Discussion by crocket** [netblue30/firejail#4532] [Q&A] [answered]
Once librewolf is sandboxed, I can't execute `librewolf --new-tab URL` to launch a new tab in the existing librewolf isntance.

My terminal emulator bookmark manager opens new tabs by executing `liberwolf --new-tab URL`.

How can I still allow `librewolf --new-tab URL` to open a new tab in sandboxed librewolf?

I want to keep a small hole open in the sandbox.

Somehow, I can open new tabs in sandboxed qutebrowser with external commands from outside sandbox.

**Comment by crocket** [comment]


**Reply by rusty-snake** [reply]
> For firefox, insert dbus-user.own org.mozilla.firefox.*

https://github.com/netblue30/firejail/blob/af947720d5d5f7b4eff5bd498c66a5b31e4e4e8a/etc/profile-a-l/firefox.profile#L51

**Comment by crocket** [comment, marked as answer]
The quick fix is to insert

```
dbus-user.own org.mozilla.librewolf.*
```

into librewolf.local. But, it introduces another issue which will be discussed in another thread.
